docs/getting-started/quickstart.md

```diff
@@ -0,0 +1,185 @@
+---
+slug: quickstart
+append_help_link: true
+title: Quickstart
+hide_title: false
+description: Learn how to set up Semgrep and scan your first repository.
+tags:
+  - quickstart
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Learn how to set up Semgrep, scan your first project for security issues, and view your findings.
+
+:::info prerequisites
+You must have Python 3.8 or later installed on the machine where the Semgrep CI is running.
+:::
+
+1. Navigate to [Semgrep](https://semgrep.dev/login), and sign up by clicking on **Sign in with GitHub** or **Sign in with GitLab**. Follow the on-screen prompts to grant Semgrep the needed permissions and proceed.
+
+2. Provide the **Organization display name** you'd like to use, then click **Create new organization**.
+
+3. Launch your CLI, and follow the instructions on the [**Scan a project on your machine**](https://semgrep.dev/onboarding/scan) page. For your convenience, the same information is presented below, along with instructions for Windows users.
```

Comment:
Just curious - couldn't we have started from the CLI? If you type in `semgrep login`, it would have opened the login page without the user having to navigate to it, and prompted the user to create an org, and the user would have "expected" to go back to the CLI at some point. Right now, we start with the online app, then move them to the CLI. Is this what happened with the onboarding tour, too? Seems weird that we bring them out of the app out of nowhere.

Reply:
I am resolving as this is as intended by the product folks. Thank you for bearing with me!